examples/table.rs

```diff
@@ -7,17 +7,32 @@ use crossterm::{
 };
 use itertools::Itertools;
 use ratatui::{prelude::*, widgets::*};
+use style::palette::material;
+
+const EXAMPLE_TABLE_COLORS: [material::AccentedPalette; 4] = [
+    material::RED,
+    material::GREEN,
+    material::BLUE,
+    material::ORANGE,
+];
+const INFO_TEXT: &str =
+    "(q) to quit / (j, ↑) to go up / (k, ↓) to go down / (l, →) for next color / (h, ←) for previous color";
```

Comment:
Oh - I should have written why on this:
l is too easy to confuse for 1 an i, so caps for shortcuts makes sense
Making this narrower is useful because our demos often tend to be rendered into around 78 chars wide.

Reply:
I agree that we should make the just `ESC` and arrow keys, and hide `qhjkl` from messages. Terminal / vim users will naturally try it, so it is enough if the examples support it.

Reply:
Now it looks like this

```
const INFO_TEXT: &str =
    "(Esc) quit | (↑) move up | (↓) move down | (→) next color | (←) previous color";
```

<img width="840" alt="Screenshot 2024-01-18 at 16 00 06" src="https://github.com/ratatui-org/ratatui/assets/30180366/47b4a1b5-66de-437f-98e6-e02580417261">